modules/cluster-logging-exported-fields-kubernetes.adoc

```diff
@@ -1,60 +1,385 @@
+
+// DO NOT MODIFY THE FOLLOWING CONTENT. Instead, update https://github.com/ViaQ/documentation/blob/main/src/data_model/model/kubernetes.yaml and run `make` as instructed here: https://github.com/ViaQ/documentation
+
+The namespace for kubernetes-specific metadata.
+
+
+=== kubernetes.pod_name
+
+The name of the pod
+
+[horizontal]
+Data type:: keyword
+
+
+
+
+
+=== kubernetes.pod_id
+
+The Kubernetes ID of the pod
+
+[horizontal]
+Data type:: keyword
+
+
+
+
+
+=== kubernetes.namespace_name
+
+The name of the namespace in Kubernetes
+
+[horizontal]
+Data type:: keyword
+
+
+
+
+
+=== kubernetes.namespace_id
+
+The ID of the namespace in Kubernetes
+
+[horizontal]
+Data type:: keyword
+
+
+
+
+
+=== kubernetes.host
+
+The Kubernetes node name
+
+[horizontal]
+Data type:: keyword
+
+
+
+
+
+
+
+
+=== kubernetes.container_name
+
+The name of the container in Kubernetes
+
+[horizontal]
+Data type:: keyword
+
+
+
+
+
+
+
+=== kubernetes.annotations
+
+Annotations associated with the Kubernetes object
+
+[horizontal]
+Data type:: group
+
+
+
+
+
+=== kubernetes.labels
+
+Labels attached to the Kubernetes object. Each label name is a subfield of labels field. Each label name is de-dotted: dots in the name are replaced with underscores.
+
+[horizontal]
+Data type:: group
+
+
+
+
+==== kubernetes.labels.deployment
+
+The deployment associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `logging-kibana-3`
+
+
+
+
+==== kubernetes.labels.deploymentconfig
+
+The deployment configuration associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `logging-kibana`
+
+
+
+
+
+
+==== kubernetes.labels.component
+
+The component associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `kibana`
+
+
+
+
+==== kubernetes.labels.provider
+
+The provider associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `openshift`
+
+
+
+
+
+=== kubernetes.event
+
+The Kubernetes event obtained from the Kubernetes master API. The event is a whole JSON object nested under the `kubernetes` field. This event description loosely follows `type Event` in link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#event-v1-core[Event v1 core].
+
+[horizontal]
+Data type:: group
+
+
+
+
+==== kubernetes.event.verb
+
+The type of event: `ADDED`, `MODIFIED`, or `DELETED`
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `ADDED`
+
+
+
+
+==== kubernetes.event.metadata
+
+Information related to the location and time of the event creation
+
+[horizontal]
+Data type:: group
+
+
+
+
+===== kubernetes.event.metadata.name
+
+The name of the object that triggered the event creation
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `java-mainclass-1.14d888a4cfc24890`
+
+
+
+
+===== kubernetes.event.metadata.namespace
+
+The name of the namespace that induced the event. This is different from `namespace_name`.
```

Comment:
@alanconway: Can you help me rewrite the last phrase, in italics. I'm not sure what we mean to say. The original in the yaml file reads:         "description: The name of the namespace that induced the event. _It differs from `namespace_name`, which will be in case of every event the 'eventrouter'._ "

Reply:
I'm not sure that is correct. Lets mark the event section  private for now, I was unsure about including it and there are some questions about whether we want to keep supporting the eventrouter. I'll review that section, we can re-public it in a future release, I'd rather err on the side of not publishing incorrect info.

Reply:
Marked it private.